packages/browser/src/browser/standalone.ts

```diff
@@ -71,6 +72,16 @@ async function attempt<T>(promise: () => Promise<T>) {
   }
 }
 
+const globalAnalyticsKey = (
+  document.querySelector(
+    'script[data-global-segment-analytics-key]'
+  ) as HTMLScriptElement
+)?.dataset.globalSegmentAnalyticsKey
+
+if (globalAnalyticsKey) {
+  setGlobalAnalyticsKey(globalAnalyticsKey)
```

Comment:
This was a good idea to live here, so it doesn't get included in the npm bundle.